### DISCUSSION TITLE
Where is the list of excluded sites for "block large media option" stored?

### DISCUSSION THREAD

**Discussion by vvkdev** [uBlockOrigin/uBlock-issues#2023] [Q&A]
How can I backup and restore this list? Backup and restore settings by the txt file doesn't help.
p.s. sorry for my bad English.

**Comment by ghost** [comment]
In "My rules" tab. 

Some info here: https://github.com/uBlockOrigin/uBlock-issues/wiki/Per-site-switches#no-large-media-elements

**Reply by vvkdev** [reply]
Thanks, but there are no new rules added in this tab and there are no my excluded sites respectively.
I have read the information on the link, there are no instructions about my question.
And if this tab is used, why does the export / import not work?

**Reply by ghost** [reply]
![image](https://user-images.githubusercontent.com/886325/156856468-21b65500-ba09-4480-954c-43e4bbb0b875.png)

Exporting works, but I see an issue - only filtered content is exported. There was something similar happening with cloud storage before. @gorhill ?

**Reply by vvkdev** [reply]
Sorry, I found that I didn't save my rules as permanent, so they didn't export. Thank you!
